src/main/java/cubicchunks/world/column/CubeMap.java

```diff
@@ -208,6 +204,6 @@ private int binarySearch(int cubeY) {
 			}
 		}
 
-		return mid; // not found :(
+		return start; // not found :(
```

Comment:
When I created this method, I remember thinking about using an existing one...
Maybe I could not find one that worked well when returning the index of where a non existent element should be.
I will re-look into this when I write tests.